test/integration/admin_login_test.rb

```diff
@@ -137,14 +137,15 @@ def setup
                              last_name: 'Doe',
                              email: 'unique@yahoo.com',
                              role: 0)
-    @order = Order.create(total_cost: 100, user_id: @non_admin.id)
+    @order = Order.create(total_cost: 100, user_id: @non_admin.id, status: "completed")
     @item = @order.items.create(title: 'coffee', description: 'black nectar of the gods', price: 1200)
 
     visit root_url
 
     click_link_or_button('Admin Dashboard')
     click_link_or_button('Orders')
-    within('#admin-order-table') do
+
+    within('#completed') do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.